server/src/main/java/org/elasticsearch/action/admin/cluster/stats/CCSUsage.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.admin.cluster.stats;
+
+import org.elasticsearch.core.TimeValue;
+
+import java.util.Map;
+
+/**
+ * This is a snapshot of telemetry from an individual cross-cluster search for _search or _async_search (or
+ * other search endpoints that use the TransportSearchAction such as _msearch).
+ */
+public class CCSUsage {
+    private final long took;
+    private final String failureType;  // TODO: enum?
```

Comment:
Since we're dealing with multiple clusters, there could be a number of failures, and partial failures. Should we have more here than a single string? Is it useful for anything?